templates/vue/src/components/Lightbox/media/MultiContentMedia/MultiContentRow.vue

```diff
@@ -1,30 +1,33 @@
 <template>
   <b-col :class="htmlClass" cols="12" :lg="node.typeData.halfWidth ? 6 : 12">
-    <div v-if="presentationStyle === 'accordion'" class="button-row">
-      <button class="trigger-row-btn" :disabled="disabled" @click="toggle">
-        <i v-if="!disabled" :class="isVisible ? 'fas fa-minus' : 'fas fa-plus'"></i>
-        <i v-else class="fas fa-lock fa-sm title-row-ico"></i>
-        {{ node.title }}
-        <locked-content
-          v-if="disabled"
-          :node="node"
-          :condition-node="conditionNode"
-        ></locked-content>
-      </button>
-      <a v-if="canEditNode" @click="editNode">
-        <i class="fas fa-pencil-alt fa-sm pr-2"></i>
-      </a>
-      <a
-        v-if="!disabled"
-        class="favourite-btn"
-        :class="{ 'is-favourite': isFavourite(node.id) }"
-        @click="toggleFavourite(node.id)"
-      >
-        <i class="fas fa-heart fa-sm"></i>
-      </a>
-    </div>
-    <div v-if="isVisible" class="multi-content-row">
-      <div v-if="presentationStyle !== 'accordion'" class="title-row-icon">
+    <div class="multi-content-row">
+      <div v-if="presentationStyle === 'accordion'" class="button-row">
+        <button class="trigger-row-btn" :disabled="disabled" @click="toggle">
+          <i
+            v-if="!disabled"
+            :class="isVisible ? 'fas fa-minus' : 'fas fa-plus'"
+          ></i>
+          <i v-else class="fas fa-lock fa-sm title-row-ico"></i>
```

Comment:
@legendword I think the class `title-row-ico` here is misspelled? (missing an `n`).. I know it's not part of the changed code, but can we fix this or remove the class name otherwise if not needed?

Reply:
I think it is being used, so I fixed the typo.